dataset-cellsystems-2021/fish/gene_pairs.json

```diff
@@ -0,0 +1,42 @@
+[
+    {
+        "GenePairName": "MYH6-MYH7",
+        "Channels": "561-638"
+    },
+    {
+        "GenePairName": "HPRT1-COL2A1",
+        "Channels": "561-638"
+    },
+    {
+        "GenePairName": "ATP2A2-H19",
+        "Channels": "561-638"
+    },
+    {
+        "GenePairName": "TCAP-BAG3",
+        "Channels": "561-638"
+    },
+    {
+        "GenePairName": "NKX25-CNTN5",
+        "Channels": "561-638"
+    },
+    {
+        "GenePairName": "MEF2C-MYL7",
+        "Channels": "561-638"
+    },
+    {
+        "GenePairName": "PLN-PRSS35",
+        "Channels": "561-638"
+    },
+    {
+        "GenePairName": "BMPER-VCAN",
+        "Channels": "561-638"
+    },
+    {
+        "GenePairName": "ACTN2-TTN",
+        "Channels": "561-638"
+    },
+    {
+        "GenePairName": "TTN-ACTN2",
+        "Channels": "561-638"
```

Comment:
@tanyasg  in the long run we will have a better way of dealing with this, but for now, these should look like this: 
```    "StructureId/Name": "TTN-ACTN2",
        "ProteinId/DisplayName": "TTN-ACTN2",
        "CellLineId/Name": "TTN-ACTN2",
        "GeneId/Name": "TTN-ACTN2",
        "GeneId/FullName": "TTN-ACTN2",
        "ProteinId/Name": "TTN-ACTN2",
        "Channels": "561-638"
```

Reply:
And "TTN-ACTN2" should be in the `file_info` where cell line normally goes in cell_feature_analysis json